runtime/doc/options.txt

```diff
@@ -5159,8 +5159,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	unescaping, so to keep yourself sane use |:let-&| like shown above.
 							*shell-powershell*
 	To use powershell (on Windows): >
-		set shell=powershell shellquote=( shellpipe=\| shellredir=> shellxquote=
+		set shell=powershell shellquote=( shellpipe=\| shellxquote=
 		set shellcmdflag=-NoLogo\ -NoProfile\ -ExecutionPolicy\ RemoteSigned\ -Command
+		set shellredir=\|\ Out-File\ -Encoding\ ASCII
```

Comment:
would this be a problem for utf-8?

Reply:
https://www.w3.org/2001/06/utf-8-test/UTF-8-demo.html
`ASCII` encoding corrupts the file.

```ps1
Get-Content -Encoding UTF8 UTF-8-demo.html | Out-File foo.txt
```
foo.txt seems to retain the utf-8 characters if omitting `-Encoding` or using `utf8`, `utf32`, `unicode` encoding values.